main.cpp

```diff
@@ -37,6 +37,36 @@ int main() {
     std::cout << simplex.find_solution();
 }
 
```

Comment:
Overal — let's follow for this time for snake_case for method and variables names